packages/components/src/MDXDeck.js

```diff
@@ -9,13 +9,15 @@ import defaultTheme from '@mdx-deck/themes/base'
 import Provider from './Provider'
 import Slide from './Slide'
 import Presenter from './Presenter'
+import Mobile from './Mobile'
```

Comment:
Just starting to dig in here, but I'd kinda prefer to avoid platform-specific code – I'm guessing what you've done would be applicable for iPads as well, right?

Reply:
It is just another mode which gets active if you open up with a query `mode=mobile`